lib/sub-tasks/git.js

```diff
@@ -0,0 +1,39 @@
+import isAdded from 'is-git-added';
+import isGit from 'is-git-repository';
+import Listr from 'listr';
+
+import getCommits from '../helpers/getCommits';
+
+const cleanup = () => {
+  const cwd = process.cwd();
+  const commits = getCommits();
+
+  return new Listr([
+    {
+      title: 'check if this directory is a git repository',
+      task: () => {
+        if (!isGit(cwd)) {
```

Comment:
Do you still need to check this and the next? It is already checked before right?